### PR TITLE
Ad Tracking: ruliweb.com

### PR DESCRIPTION
**Pull request by ghost**
### URL
https://www.ruliweb.com/

### Screenshot
hhh

### Network Log
scre3

### Environment
Browser: Mozilla Firefox Developer Edition 92.0b3
Filter List: `List-KR` `AG Base` `AG Mobile Ad` `AG TPF` `AG URL TF` `Fanboy Anti-3rd Font` `Fanboy Anti-FB` `AG Social` `AG Annoyances` `AG Chinese` `AG Dutch` `AG French` `AG German` `AG Japanese` `AG Russian` `AG Spanish` `AG Turkish`

### CODE REVIEW

File: EnglishFilter/sections/adservers_firstparty.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 !
 ! Section contains the list of advertising networks, which are hosted on non advertising sites as subdomains
 !
+||api.adhooah.com^
```
Comment (zloyden) [inline EnglishFilter/sections/adservers_firstparty.txt:4]: Why not `||adhooah.com^$third-party`?

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -4360,6 +4360,8 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/91114
+ruliweb.com###pc_maintop_hotdeal
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:4364]: There are missed leftovers
```
ruliweb.com##div[id^="ad_main"]
ruliweb.com#?#.main_aside > div.aside_background + div[id^="ad_main"]:upward(1)
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 2 above)
Comment (piquark6046) [reply EnglishFilter/sections/foreign.txt:4364]: https://github.com/AdguardTeam/AdguardFilters/pull/91114/commits/7e739e1e438eca0c3a5b3320a266182c0d1372c0

File: EnglishFilter/sections/adservers_firstparty.txt (hunk 1 above)
Comment (piquark6046) [reply EnglishFilter/sections/adservers_firstparty.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/91114/commits/7e739e1e438eca0c3a5b3320a266182c0d1372c0

**Review by zloyden — APPROVED** [review]

**Pull request merged**